Project21/Project21/Testcase.c

```diff
@@ -0,0 +1,77 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.Naveenkumar s
+// ------------------------------------------------------------------
+// Testcase.c
+// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+// ------------------------------------------------------------------------------------------------
+#include"conversionfile.h"
+
+void PrintResult (int input[], char* output[], char* (*func)(int), int length) {
+   for (int i = 0; i < length; i++) {
+      printf ("TestCase%2d: input->%12d ", i + 1, input[i]);
+      printf ((strcmp (func (input[i]), output[i]) == 0) ? ("%11s", "pass\n") : (" %11s", "Fail\n"));
+   }
+}
+
+int main () {
+   int arr[] = { -255,65535, 5125412, -592,859,-7587,865482,0,-1784587,INT_MIN,-2147483649ll };

```

Comment:
```suggestion
   int arr[] = { -255,65535, 5125412, -592,859,-7587,865482,0,-1784587,INT_MIN,-2147483649};

```
You are assigning a long long value to an int[] array. This is invalid.